pandas/tests/resample/test_base.py

```diff
@@ -98,11 +98,13 @@ def test_raises_on_non_datetimelike_index():
 
 @all_ts
 @pytest.mark.parametrize("freq", ["M", "D", "H"])
-def test_resample_empty_series(freq, empty_series_dti, resample_method):
+def test_resample_empty_series(freq, empty_series_dti, resample_method, request):
     # GH12771 & GH12868
 
-    if resample_method == "ohlc":
-        pytest.skip("need to test for ohlc from GH13083")
```

Comment:
is the 13083 reference no longer relevant?

Reply:
It's a closed issue, but I'll add back the reference.